src/excalidraw-utils/webpack.prod.config.js

```diff
@@ -0,0 +1,40 @@
+const webpack = require("webpack");
+const path = require("path");
+
+module.exports = {
+  mode: "production",
+  resolve: {
+    extensions: [".tsx", ".ts", ".js"],
```

Comment:
`.tsx` won't be needed i guess since these will be pure functions ?

Reply:
Actually, I cannot remove `.tsx` because some components are "leaking" in the require/import chains.

For instance, we are importing some functions from `src/element/index.ts` which import `export { textWysiwyg } from "./textWysiwyg";`. Not sure why but `textWysiwyg` has a `.tsx` extension.